src/GeneralStateTestsFiller/stEWASMTests/logFiller.yml

```diff
@@ -0,0 +1,139 @@
+              ;; topic3 i32ptr the memory offset to load topic3 from (u256)
+              (i32.const 0)
+              ;; topic4 i32ptr the memory offset to load topic4 from (u256)
+              (i32.const 0)
+            )
+          )
+        )
+      nonce: ''
+      storage: {}
+    # main contract, tx receiver
+    abcdabcdabcdabcdabcdabcdabcdabcdabcdabcd:
+      balance: '100000000000'
+      code: |
+        (module
+          (import "ethereum" "call" (func $call (param i64 i32 i32 i32 i32) (result i32)))
+          (import "ethereum" "storageStore" (func $storageStore (param i32 i32)))
+          (import "ethereum" "getReturnDataSize" (func $getReturnDataSize (result i32)))
+          (import "ethereum" "returnDataCopy" (func $returnDataCopy (param i32 i32 i32)))
+
+          (memory 1)
+          (data (i32.const 0)  "\de\ad\be\ef\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00")
+          ;; storage keys
+          (data (i32.const 128) "\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00") ;; 0
+          (data (i32.const 160) "\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\01") ;; 1
+          (export "memory" (memory 0))
+          (export "main" (func $main))
+
+          (func $main
+            ;; memory layout
+            (local $memAddress i32)
+            (local $memValue i32)
+            (local $memRetdata i32)
+            (local $memResult i32)
+            (local $memStoragekey1 i32)
+            (local $memStoragekey2 i32)
+            (local $memRetdatasize i32)
+
+            (set_local $memAddress (i32.const 0))
+            (set_local $memValue (i32.const 32))
+            (set_local $memResult (i32.const 96))
+            (set_local $memStoragekey1 (i32.const 128))
+            (set_local $memStoragekey2 (i32.const 160))
+            (set_local $memRetdatasize (i32.const 224))
+
+            (i32.store
+              ;; store success/failure in mem
+              (get_local $memResult)
+              (call $call
+                ;; gas i64 the gas limit
+                (i64.const 100000)
+                ;; addressOffset i32ptr the memory offset to load the address from (address)
+                (get_local $memAddress)
+                ;; valueOffset i32ptr the memory offset to load the value from (u128)
+                (get_local $memValue)
+                ;; dataOffset i32ptr the memory offset to load data from (bytes)
+                (get_local $memValue)
+                ;; dataLength i32 the length of data
+                (i32.const 1)
+              )
+            )
+
+            ;; Store return data size to check it (it should be 0)
+            (i32.store (get_local $memRetdatasize) (call $getReturnDataSize))
+            
+            (call $storageStore (get_local $memStoragekey1) (get_local $memRetdatasize))
+
+            ;; Store the return value to make sure it was 2 (revert)
```

Comment:
The result of the contract call is zero, it is not reverting.

Reply:
Why should it revert? Sorry if I'm missing something obvious here. I don't think this is intended to fail, it's just testing gas usage.

Reply:
The code comment is incorrect.

Reply:
@lrettig @jwasinger right, it is just the comment